### DISCUSSION TITLE
Transfer data over the wfb_tx/rx link (images)

### DISCUSSION THREAD

**Discussion by XBeg9** [svpcom/wfb-ng#278] [Questions] [answered]
I see that you are not recommending using the `tunnel` channel for any massive binary transfer, but curious if I just need to transfer binary data like images over the link, what you will recommend, should be different from the video link?

Thanks!

**Comment by svpcom** [comment, marked as answer]
tunnel (with default fec 1/2) is not recommended for regular transfer, but you can use it for maintenance task. If you use if for regular tasks then you can set fec 8/12 and add fec_timeout. This will increase bandwidth, but also will increase latency.

**Reply by XBeg9** [reply]
It sounds better to have separate wfb_tx/rx just for this case and keep default values for the tunnel. Thanks @svpcom

**Reply by svpcom** [reply]
yes
